front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -251,7 +255,9 @@ const PacedTrainItem = ({
         <TimetableItemActions
           selectPathProjection={selectPathProjection}
           duplicateTimetableItem={duplicatePacedTrain}
-          editTimetableItem={() => selectPacedTrainToEdit(pacedTrain)}
+          // The first pacedTrain is the one we use to update it
+          // The second one is the one we use to pass down the exceptions
+          editTimetableItem={() => selectPacedTrainToEdit(pacedTrain, pacedTrain)}
```

Comment:
When you edit the pacedTrain, I think you don't need to give the original one
```suggestion
          editTimetableItem={() => selectPacedTrainToEdit(pacedTrain)}
```

Reply:
Done